### PR TITLE
Solves #251: add a note to the doc on 63 chars limit

### PR DESCRIPTION
**Pull request by tracek**
Trivial change in doc to address https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/251

### CODE REVIEW

File: doc/source/user-experience.rst [hunk 1]
```diff
@@ -57,6 +57,8 @@ existing image, such as the ``scipy-notebook`` image, complete these steps:
            tag: 8e15d329f1e9
 
    .. note::
+   
+      Container name cannot be longer than 63 characters.
```
Comment (yuvipanda) [inline doc/source/user-experience.rst]: This should probably say that the container *image* name can not be longer than 63 characters?

Comment (tracek) [reply doc/source/user-experience.rst]: Indeed, thanks. Fixed.

**Comment by yuvipanda** [comment]
Thanks for the patch, @tracek! Welcome to the community, and I hope to see more patches from you 'round here :)

**Comment by yuvipanda** [comment]
Test failures are unrelated, since this is a doc change.

**Pull request merged**